samples/HostConfig/viva-connections-light.json

```diff
@@ -67,8 +67,8 @@
 					"subtle": "#E1DFDD"
 				},
 				"accent": {
-					"default": "#6264A7",
-					"subtle": "#A6A7DC"
+					"default": "#5B5FC7",
+					"subtle": "#C5CBFA"
```

Comment:
Why do we have to make changes in this file??
Intuitively, light theme shouldn't be altered by the introduction of a dark theme.

Reply:
Great question - When I went in search of the Teams dark theme palette, I discovered that there are new primary colors for the light palette as well. Updated them in this PR to be consistent.